src/main/java/nl/hsac/fitnesse/fixture/util/selenium/SeleniumHelper.java

```diff
@@ -967,6 +967,10 @@ public void doubleClick(WebElement element) {
         getActions().doubleClick(element).perform();
     }
 
+    public void clickWithOffset(WebElement element,int xOffset,int yOffset) {
+        getActions().moveToElement(element, xOffset, yOffset ).click().build().perform();
```

Comment:
No need to call build, you can just call `perform()` after click.

I now see that the offset is relative to top left hand corner of element. That needs to be in documentation...